inc/compat/themes/compat-theme-blocksy.php

```diff
@@ -19,10 +19,288 @@ public function __construct() {
+
+		$blocksy = Blocksy\Plugin::instance();
+
+		// Bail if header property is not available
+		if ( ! isset( $blocksy->header ) || ! is_object( $blocksy->header ) ) { return $attributes; }
+
+		$blocksy_header = $blocksy->header;
+
+		// Bail if method is not available
+		if ( ! method_exists( $blocksy_header, 'current_screen_has_sticky' ) ) { return $attributes; }
+
+		// Get sticky header settings from Blocksy
+		$sticky_header_settings = $blocksy_header->current_screen_has_sticky();
+
+		// Bail if theme's conditions for sticky header are not met
+		if ( ! $sticky_header_settings || ! is_array( $sticky_header_settings ) ) { return $attributes; }
+
+		// Bail if no devices are selected for sticky header
+		if ( ! isset( $sticky_header_settings['devices'] ) || empty( $sticky_header_settings['devices'] ) ) { return $attributes; }
+
+		// Default values
+		$mobile_settings = '';
+		$desktop_settings = '';
+
+		// Set the sticky header settings based on the selected devices
+		if ( in_array( 'desktop', $sticky_header_settings['devices'] ) ) {
+			$desktop_settings = '"sm": { "breakpointInitial": 1000, "breakpointFinal": 10000, "selector": "header [data-sticky*=yes]" }';
+		}
+		if ( in_array( 'mobile', $sticky_header_settings['devices'] ) ) {
+			$mobile_settings = '"xs": { "breakpointInitial": 0, "breakpointFinal": 999, "selector": "header [data-sticky*=yes]" }';
+		}
+
+		// Only keep non-empty values
+		$settings = array_filter( array( $mobile_settings, $desktop_settings ), function( $value ) {
+			return !empty( $value );
+		} );
+
+		// Concatenate values with a comma
+		$settings = implode( ', ', $settings );
+
+		$attributes['data-sticky-relative-to'] = "{ {$settings} }";
+
+		return $attributes;
+	}
+
+
+
+	/**
+	 * Add new settings to the Fluid Checkout admin settings sections.
+	 *
+	 * @param   array   $settings         Array with all settings for the current section.
+	 * @param   string  $current_section  Current section name.
+	 */
+	public function add_settings( $settings ) {
+		// Add new settings
+		$settings_new = array(
+			array(
+				'title' => __( 'Theme Blocksy', 'fluid-checkout' ),
+				'type'  => 'title',
+				'id'    => 'fc_integrations_theme_blocksy_options',
+			),
+
+			array(
+				'title'           => __( 'Color switch', 'fluid-checkout' ),
+				'desc'            => __( 'Force FC elements to follow color mode settings from the Blocksy theme.', 'fluid-checkout' ),
+				'id'              => 'fc_compat_theme_blocksy_force_color_mode_switch',
+				'type'            => 'checkbox',
+				'default'         => FluidCheckout_Settings::instance()->get_option_default( 'fc_compat_theme_blocksy_force_color_mode_switch' ),
+				'autoload'        => false,
+			),
+
+			array(
+				'type' => 'sectionend',
+				'id'    => 'fc_integrations_theme_blocksy_options',
+			),
+		);
+
+		$settings = array_merge( $settings, $settings_new );
+
+		return $settings;
+	}
```

Comment:
Remove this option from the plugin integration settings, since we should always follow the color mode set by the theme.